### PR TITLE
Adding notes about how to edit the config

### PR DESCRIPTION
**Pull request by DubhAd**
## Proposed change

It was quite reasonably pointed out that while the `configuration.yaml` file is mentioned, there's no mention of how you can reach it and edit it.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration.markdown [hunk 1]
```diff
@@ -28,6 +28,16 @@ If you run into trouble while configuring Home Assistant, have a look at the [co
 
 </div>
 
+## Editing `configuration.yaml`
+
+There are many ways you can edit `configuration.yaml`. Here are three options to get you started:
+
+The most basic is to use SSH to connect to the system (you may need to install the SSH add-on) and then use `nano` (or `vim`) to edit the file.
```
Comment (frenck) [inline source/_docs/configuration.markdown]: Should we really list this suggestion for the first one? I can see my dad already trying this... 😂

File: source/_docs/configuration.markdown [hunk 2]
```diff
@@ -28,6 +28,16 @@ If you run into trouble while configuring Home Assistant, have a look at the [co
...
+
+You can also use Samba (again, you may need to install the Samba add-on) and your favourite file editor.
+
+There's a [Visual Studio Code](https://github.com/hassio-addons/addon-vscode#home-assistant-community-add-on-visual-studio-code) that allows you to edit your configuration within Home Assistant itself.
```
Comment (frenck) [inline source/_docs/configuration.markdown]: Visual Studio Code availability is limited, maybe list the File Editor add-on that is in the core repo?

File: source/_docs/configuration.markdown (hunk 1 above)
Comment (DubhAd) [reply source/_docs/configuration.markdown]: Moved to last

File: source/_docs/configuration.markdown (hunk 2 above)
Comment (DubhAd) [reply source/_docs/configuration.markdown]: Updated ;)

**Review by frenck — DISMISSED** [review]
Thanks, @DubhAd 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**